modules/rosa-adding-node-labels.adoc

```diff
@@ -16,7 +16,7 @@ Labels are assigned as key-value pairs. Each key must be unique to the object it
 
 ifdef::openshift-rosa[]
 * You installed and configured the latest AWS (`aws`), ROSA (`rosa`), and OpenShift (`oc`) CLIs on your workstation.
-* You logged in to your Red Hat account by using the `rosa` CLI.
+* You logged in to your Red Hat account by using the {product-title} (ROSA) CLI (`rosa`).
```

Comment:
Given that all of the command line tools are mentioned with their short names in the previous line, it seems odd to expand this here. However, the rest of the procedure only seems to use the ROSA CLI so maybe we can change lines 18 and 19 to be something like:

```
* You installed and configured the latest {product-title} (ROSA) CLI (`rosa`) on your workstation.
* You logged in to your Red Hat account using the ROSA CLI.
```

Reply:
Hello @xueli181114 . This PR is focused only on making the terminology consistent and not changing any actual content in command blocks. So, this don't necessarily need to go through QE review. However, I would really appreciate your input for this comment. Could you please help? 
I can then resolve similar following comments for this PR. Thank you!

Reply:
Hello @yuwang-RH Thanks for the QE review. Based on your [approval](https://github.com/openshift/openshift-docs/pull/61151#issuecomment-1611016924) for this PR, I will be updating Lines 18 and 19 as @laubai suggested in this comment. Please let me know if you have other thoughts. Thank you!

Reply:
@smunje1 It if fine for me.